gotocompany/guardian/v1beta1/guardian.proto

```diff
@@ -416,6 +416,8 @@ message ListResourcesRequest {
     gte: 0,
     ignore_empty: true
   }];
+  repeated string order_by = 10;
+  string direction = 11;
```

Comment:
in ListAppeals, ListApprovals and ListGrants, we only provide `order_by` field and the way direction also part of the same field for example: `?order_by=created_at:asc&order_by=foobar:desc`. Can we also do the same here?